api.js

```diff
@@ -1,6 +1,6 @@
 const Sentry = require('@sentry/node')
 
-const Web3 = require('web3')
+const { ethers } = require("ethers");
```

Comment:
should be single quote `"ethers"`😂 
sorry there is no linter in the build. 
Can you run `npm run test` just to make sure linting passes

Reply:
Sure will do